.travis.yml

```diff
@@ -105,10 +104,10 @@ notifications:
 before_install:
   - eval "${MATRIX_EVAL}"
   - $CXX --version
-###  - echo "Installing pip, setuptools, and wheel"
-###  - pip install --upgrade pip setuptools wheel || exit
-###  - echo "Installing Python dependencies"
-###  - pip install -r $TRAVIS_BUILD_DIR/bindings/py/requirements.txt --only-binary --quiet || exit
+  - echo "Installing pip, setuptools, and wheel"
+  - pip install --upgrade pip setuptools wheel || exit
+  - echo "Installing Python dependencies"
+ #  - pip install -r $TRAVIS_BUILD_DIR/bindings/py/packaging/requirements.txt --only-binary --quiet || exit
```

Comment:
> why is this still commented?

setup.py already does this so this is also redundant.  However, line 110 is executed before setup.py.  So that is why I commented it out rather than just deleting it.